src/main/java/net/starschema/clouddb/jdbc/BQStatement.java

```diff
@@ -193,6 +198,18 @@ private ResultSet executeQueryHelper(String querySql, boolean unlimitedBillingBy
                   .get(projectId, qr.getJobReference().getJobId())
                   .setLocation(qr.getJobReference().getLocation())
                   .execute();
+
+          BiEngineStatistics biEngineStatistics = null;
```

Comment:
I don't see this being used at this level.  You can move the declaration in to the level where it's actually used.  That should help readability, since the reader will know that they only need to track this to that inner scope, rather than across the full function.